client/src/featureform/resources.py

```diff
@@ -216,6 +218,7 @@ def serialize(self) -> bytes:
             "Credentials": self.credentials.config(),
```

Comment:
Can we put this dict in a function so its consistent?

Reply:
the other configs have serialize method the same way. we can call `self.config()` here but we will have to change all of the other ones too to stay consistent.

Reply:
What i mean is that why doesnt serialized just call `config=self.config()` instead of rewriting it again right below